nss-day-cohort-39/tabloid-full-stack-vanjies
# Update post

### What I Added:
1. added an edit button to all posts given they are by the current user
2. made a form modal to be used if users wan to edit posts they made.
3. When edit modal form is submitted the post in the database will be updated with the new input, and the user is directed back to the list if they started there, or the post details if they started there.

### How To Test:
1. view the "Posts" and MyPosts menu options and see that there is an edit button on all post by the user you are logged in with.
2. go to the details page for a post and see that there is an edit button there as well. 
3. click the edit button on all three locations and see that they all bring up a modal populated with the post's data, and allows you to change the data and submit.
4. after submitting check that the post has changed in the lists and details views
